x-pack/plugins/dataset_quality/server/types/default_api_types.ts

```diff
@@ -13,7 +13,7 @@ export const typeRt = t.partial({
   type: dataStreamTypesRt,
 });
 
-export const rangeRt = t.type({
+export const rangeRt = t.partial({
```

Comment:
This is used in other endpoints too, do we aim to make it optional there as well?

Reply:
Maybe the default timerange could be passed from the client, and keep the server types as required